README.md

```diff
@@ -148,7 +132,7 @@ git clone --single-branch https://github.com/rmcolq/pandora.git --recursive
 cd pandora
 mkdir -p build
 cd build
-cmake ..
+cmake -DCMAKE_BUILD_TYPE=ReleaseWithAsserts ..
```

Comment:
This actually triggered me to make a deeper debug of this build type in our `CMakeLists.txt`, and I just realised that the `-O3` flag was not being passed on into `make`. So, in the end we have been simply using the normal build instead of a compiler-optimised build all the time. You can see more details about this in https://github.com/iqbal-lab/pandora1_paper/issues/245 . A nice side effect of code review! Just reverted this change on https://github.com/rmcolq/pandora/pull/243/commits/6aa846a40edc60f8b0b75ed3121c94456f20988b